File: tests/fixtures/go/simple-loop.go
// Simple loop for basic stepping and variable inspection.
// Equivalent to python/simple-loop.py.
package main

import "fmt"

func sumRange(n int) int {
	total := 0
	for i := 0; i < n; i++ {
		total += i
	}
	return total
}

func main() {
	result := sumRange(10)
	fmt.Printf("Sum: %d\n", result)
}

File: tests/fixtures/node/simple-loop.js
/**
 * Simple loop for basic stepping and variable inspection.
 * Equivalent to python/simple-loop.py.
 */
function sumRange(n) {
	let total = 0;
	for (let i = 0; i < n; i++) {
		total += i;
	}
	return total;
}

const result = sumRange(10);
console.log(`Sum: ${result}`);
